### DISCUSSION TITLE
I cannot read STEM on Mixxx

### DISCUSSION THREAD

**Discussion by rromualdi** [acolombier/stemgen#7] [Q&A] [answered]
![Screenshot 2024-11-03 094501](https://github.com/user-attachments/assets/b1c06713-c34b-4973-b9d6-6abb92c21767)
I own my stem tracks created with the Native Instrument tool, but when i load it into mixxx it is not read like a stem file, but like a classic wav or mp3 file, how do i fix it? I have installed the development snapshot.

**Comment by acolombier** [comment]
Hi @rromualdi, I suggest you reach out to the Mixxx community directly, either on Discourse or Zulip, as this repo is exclusively to create STEM from MP3 tracks. 
Looking at the `Type` column, it looks like this build doesn't have the `STEM` feature enabled.

**Reply by ezmartinez96** [reply]
Hi Acolbier, I have the same issue. I have Qt Version 6.5.3 installed. Is there any build with the Stem Mixing option enabled? or haw can I enable the option once installed? Thanks a lot.

**Reply by acolombier** [reply, marked as answer]
Hi, I assume you are also trying to get STEM working inside Mixxx. If so, please make sure to reach out to the Mixxx community. STEM should be released as part of 2.6, which should be published as beta in the next few days.

**Reply by ezmartinez96** [reply]
Thanks a lot Acolombier.

**Reply by ezmartinez96** [reply]
Question again, will the STEM feature be on the fly? or we will need to generate the stem files separately? Thanks.

**Reply by acolombier** [reply]
For now, it will need to be generated (potentially with the use of this project, stemgen). There is a GSoC proposal to introduce to implement on the fly generation, so we might see this coming in the future Mixxx version! 
Going forward, It's best to raise these questions directly to the Mixxx community (GH issues, Discourse or Zulip), as it help with visibility and popularity of feature request, which can then help us to prioritise of development effort

**Comment by ezmartinez96** [comment]
OK, thanks a lot.
